app/translations.js

```diff
@@ -73,4 +73,35 @@ export const translations = {
     errorSummary: "Erreur",
     login: "Déjà un compte ? Connectez-vous !",
   },
-}
+  petProfile: {
+    errorMessage: "Une erreur est survenue lors de la création de la fiche",
+    successMessage: "Fiche animal créée avec succès",
+    genderMale: "Mâle",
+    genderFemale: "Femelle",
+    sizeSmall: "Petit",
+    sizeMedium: "Moyen",
+    sizeBig: "Grand",
+    specieDog: "Chien",
+    specieCat: "Chat",
+    specieBird: "Oiseau",
+    specieRodent: "Rongeur",
+    specieReptile: "Reptile",
+    specieOther: "Autre",
+    submitButton: "Créer la fiche",
+    name: "Nom de l'animal",
+    specie: "Espèce",
```

Comment:
Il manque un 's' à la fin du mot :) 

```suggestion
    species: "Espèce",
```